apps/web/src/pages/get-started/components/AdditionInformationLink.tsx

```diff
@@ -0,0 +1,31 @@
+import styled from '@emotion/styled';
+
+import { colors, Text, OpenedBook } from '@novu/design-system';
+
+import { Link } from '../consts/shared';
+import { OnboardingUseCasesTabsEnum } from '../consts/OnboardingUseCasesTabsEnum';
+
+interface IAdditionInformationLinkProps extends React.AnchorHTMLAttributes<HTMLAnchorElement> {
+  channel: OnboardingUseCasesTabsEnum;
+}
+export function AdditionInformationLink({ channel, ...linkProps }: IAdditionInformationLinkProps) {
+  return (
+    <StyledLink {...linkProps}>
+      <OpenedBook />
+      <StyledText>Learn about {channel}</StyledText>
+    </StyledLink>
+  );
+}
+
+const StyledLink = styled(Link)`
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
+  margin-top: 24px;
```

Comment:
Could you elaborate a bit not sure i understand correctly.

Reply:
Since margin adds additional spacing between spacing elements, it's generally preferable to let the parent component add it:

```
...Rest of the layout...
<AdditionalInformationLink ...other props... mt={24}>...</AdditionalInformationLink>
...
<OtherComponents />
```

This way, components are more reusable